conferences/abstracts/EuroPython2020-talk.md

```diff
@@ -0,0 +1,73 @@
+# EuroPython 2020 Proposal - Talk
+
+**Where:** Online! :woman_technologist:
+
+**When:** 23rd - 26th July 2020
+
+**Conference website:** <https://ep2020.europython.eu/>
+
+**Call for Proposals website:** <https://ep2020.europython.eu/call-for-proposals/>
+
+## Title
+
+Sharing Reproducible Python Environments with Binder
+
+## Subtitle
+
+(_Optional_)
+
+## Abstract (long)
+
+_At most 1500 characters. Current count (with spaces): 1182_
+
+As reproducibility gains traction in the data science community, the need to package code, data and the computational environment is growing.
```

Comment:
You could successfully argue it's not just growing in data science, but in all (scholarly) research. That might widen your audience, who may be making their go/no-go decision on the first para.